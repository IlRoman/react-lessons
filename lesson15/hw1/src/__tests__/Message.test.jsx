import React from "react";
import Message from '../Message';
import { shallow } from 'enzyme';

describe('<Message />', () => {
    it('should render text', () => {
        const props = { text: 'Hello world' }
        const wrappedComponent = shallow(<Message {...props} />);
        expect(wrappedComponent).toMatchSnapshot();
    }),

        it('should not render text', () => {
            const props = { text: '' }
            const wrappedComponent = shallow(<Message {...props} />);
            expect(wrappedComponent).toMatchSnapshot();
        })

})