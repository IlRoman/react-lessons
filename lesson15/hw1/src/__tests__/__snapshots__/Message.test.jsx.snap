// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Message /> component should hide - no text passed 1`] = `""`;

exports[`<Message /> should not render text 1`] = `""`;

exports[`<Message /> should render text passed: Hello world 1`] = `
<div
  className="message"
>
  Hello world
</div>
`;

exports[`<Message /> should render text: Hello world 1`] = `
<div
  className="message"
>
  Hello world
</div>
`;
